chimney/src/test/scala/io/scalaland/chimney/ErrorPathSpec.scala

```diff
@@ -0,0 +1,144 @@
+package io.scalaland.chimney
+
+import io.scalaland.chimney.dsl._
+import io.scalaland.chimney.internal.utils.EitherUtils._
+import io.scalaland.chimney.utils.OptionUtils._
+
+import utest._
+
+object ErrorPathSpec extends TestSuite {
+  val tests = Tests {
```

Comment:
Added error path node for override fields and test for withFieldConstT and withFieldComputedT usage

Reply:
Hmm, this is not what I meant. I don't think that we need to treat overrides in any special way. The error path `foo.overrides(bar)` seems to be kind of misleading and I'm not really sure if I understand what does it mean. Would you mind reverting that?

What I had in mind was to add the test case that uses [transformer definition DSL](https://scalalandio.github.io/chimney/transformers/own-transformations.html#transformer-definition-dsl) for lifted transformers, so that:
```scala
Transformer
   .defineF[..., ...]
   ...
   .buildTransformer
```

which interacts with error paths support is tested for correct behavior.

Reply:
Reverted overrides error path capturing. Updated test

Reply:
Great, thank you 👍